docker_explorer/lib/storage.py

```diff
@@ -130,3 +142,137 @@ def Mount(self, container_object, mount_dir):
+
+    Args:
+      container_object (Container): the container object to mount.
+      mount_dir (str): the path to the target mount point.
+
+    Returns:
+      list: a list commands that needs to be run to mount the container's view
+        of the file system.
+    """
+    if not os.path.isfile('/sbin/mount.aufs'):
+      print('Could not find /sbin/mount.aufs. Please install the aufs-tools '
+            'package.')
+
+    mount_id = container_object.mount_id
+
+    container_layers_filepath = os.path.join(
+        self.docker_directory, self.STORAGE_METHOD, 'layers', mount_id)
+    container_id = os.path.join(
+        self.docker_directory, self.STORAGE_METHOD, 'diff', mount_id)
+    if self.docker_version == 1:
+      container_layers_filepath = os.path.join(
+          self.docker_directory, self.STORAGE_METHOD, 'layers', container_id)
+
+    commands = []
+    mountpoint_path = os.path.join(
+        self.docker_directory, self.STORAGE_METHOD, 'diff', container_id)
+    commands.append('mount -t aufs -o ro,br={0:s}=ro+wh none {1:s}'.format(
+        mountpoint_path, mount_dir))
+    with open(container_layers_filepath) as container_layers_file:
+      layers = container_layers_file.read().split()
+      for layer in layers:
+        mountpoint_path = os.path.join(
+            self.docker_directory, self.STORAGE_METHOD, 'diff', layer)
+        commands.append(
+            'mount -t aufs -o ro,remount,append:{0:s}=ro+wh none {1:s}'.format(
+                mountpoint_path, mount_dir))
+
+    commands.extend(self._MakeExtraVolumeCommands(container_object, mount_dir))
+
+    return commands
+
+
+class OverlayStorage(BaseStorage):
+  """This class implements OverlayFS storage specific methods."""
+
+  STORAGE_METHOD = 'overlay'
+  LOWER_NAME = u'lower-id'
+  UPPER_NAME = u'upper'
+
+  def _BuildLowerLayers(self, lower):
+    """Builds the mount command option for the 'lower' directory.
+
+    Args:
+      lower (str): the path to the lower directory.
+
+    Returns:
+      str: the mount command option for thr 'lower' directory.
+    """
+    return os.path.join(
+        self.docker_directory, self.STORAGE_METHOD, lower.strip(), 'root')
+
+  def MakeMountCommands(self, container_object, mount_dir):
+    """Generates the required shell commands to mount a container's ID.
+
+    Args:
+      container_object (Container): the container object.
+      mount_dir (str): the path to the target mount point.
+
+    Returns:
+      list: a list commands that needs to be run to mount the container's view
+        of the file system.
+    """
+    mount_id_path = os.path.join(
+        self.docker_directory, self.STORAGE_METHOD, container_object.mount_id)
+
+    with open(os.path.join(mount_id_path, self.LOWER_NAME)) as lower_fd:
+      lower_dir = self._BuildLowerLayers(lower_fd.read())
+    upper_dir = os.path.join(mount_id_path, self.UPPER_NAME)
+    work_dir = os.path.join(mount_id_path, 'work')
+
+    cmd = (
```

Comment:
For better readability, I would split this into two lines, one with the string into a variable and the other one with `variable.format()`

Reply:
like so?